tests/python/kaolin/spc/avg_pool_test.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import division
```

Comment:
test structure should follow directory tree of core library, the name of the python file have to start with `test_`

I only see the file `kaolin/ops/spc/spc.py` so it should be `tests/python/kaolin/spc/test_spc.py`